apps/site-app/src/containers/MainNavigation/styles.module.less

```diff
@@ -1,9 +1,97 @@
+@import "~@thecointech/site-semantic-theme/variables.less";
+
+
 .mainMenu{
     width: auto;
     display: inline-block;
     height: 40px;
     margin-bottom: 50px;
     .ui.secondary.pointing.menu{
-        border-bottom: none;   
+        border-bottom: none;
     }
-}
\ No newline at end of file
+}
+
+
+
+/** linear interpolate value from small to big as screen goes from small-size to big-size */
+.scalable-unit(@property, @small, @big, @unit) {
```

Comment:
I have seen this somewhere else I think. Is it something that can be put in the shared or base?

Reply:
yeah, I just need to do a bit more work to generalize it.  I was just investigating here